src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,60 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+
+  let result = [];
+  let consecutiveCharIndex = 1;
+  let numberDuplicate = 0;
+  let duplicateStr = '';
+
+  if(text === null || typeof text !== 'string'){
+    throw('Please enter a valid string');
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.